### PR TITLE
Mayasse bee8843

### PR DESCRIPTION
**Pull request by mayasse**

### CODE REVIEW

File: dsl/com/electriccloud/commander/dsl/util/BaseObject.groovy [hunk 1]
```diff
@@ -371,28 +417,36 @@ abstract class BaseObject extends DslDelegatingScript {
...
   }
 
-  def loadAcls (File aclDir, String objPath,
-                Map bindingMap) {
-   println "Entering loadAcls"
-   println "  dir  : $aclDir"
-   println "  path : $objPath"
-   println "  map  : " + bindingMap.toMapString(250)
+  def loadAcls (File aclDir, String objPath, Map bindingMap, changeList = [:]) {
+//    println "Entering loadAcls"
```
Comment (itykhan) [inline dsl/com/electriccloud/commander/dsl/util/BaseObject.groovy:427]: I don't see we use changeList inside the method

File: dsl/procedures/importDslFromGitNew/procedure.dsl [hunk 2]
```diff
@@ -13,14 +13,29 @@ procedure procName, {
             subprocedure: 'PullClone',
             subproject:'/plugins/EC-Git/project',
             resourceName: '$[/myJob/usedResource]',
+            description: """
+                EC-DslDeploy spec tests rely on the value of [Run]condition.
+                Do not change [Run]condition without successfully running those tests.
+            """,
+            condition: '$[/javascript (getProperty("repoUrl")!="skip_checkout_changes_step_please");]',
```
Comment (itykhan) [inline dsl/procedures/importDslFromGitNew/procedure.dsl:20]: Is such condition value just for spec tests?

File: dsl/procedures/importDslFromGitNew/steps/deleteObjectsInChangeList.groovy [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+println "Incremental Import: $incremental"
+
+def changeListText = "";
+if (incremental) {
+//    Gather change list text
+//    def fileName = "[dest]/change_list.json"
+//    // if file exists, is not a folder and is readable...
+//    def changeListFile = new File(fileName)
+//    if (changeListFile.exists() && changeListFile.isFile()) {
+//        changeListText = changeListFile.text
```
Comment (itykhan) [inline dsl/procedures/importDslFromGitNew/steps/deleteObjectsInChangeList.groovy:44]: do we need this commented out code here?

File: dsl/com/electriccloud/commander/dsl/util/BaseObject.groovy (hunk 1 above)
Comment (mayasse) [reply dsl/com/electriccloud/commander/dsl/util/BaseObject.groovy:427]: I just added it

File: dsl/procedures/importDslFromGitNew/steps/deleteObjectsInChangeList.groovy (hunk 3 above)
Comment (mayasse) [reply dsl/procedures/importDslFromGitNew/steps/deleteObjectsInChangeList.groovy:44]: We probably do not need the code.  There are 2 ways to retrieve the change list: into a property or into a file.  We chose "into a property" for importDslFromGitNew" to simplify the calling for users.

File: specs/src/test/groovy/com/electriccloud/plugin/spec/overwrite_installProject.groovy [hunk 4]
```diff
@@ -1399,21 +1399,21 @@ class overwrite_installProject extends PluginTestHelper {
         def ec_1 = projPropResult.propertySheet.property.find{it.propertyName == 'ec_1'}
 
         def ec_1Res = dsl "getProperties propertySheetId: '$ec_1.propertySheetId'"
-        assert ec_1Res.propertySheet.property.size() == 2
+        assert ec_1Res.propertySheet.property.size() == 3
```
[the remark this answers is not in the document]
Comment (mayasse) [reply specs/src/test/groovy/com/electriccloud/plugin/spec/overwrite_installProject.groovy:1402]: It appears to be a 10.5 issue, but we have not yet taken the time to go back and see where a change in behavior happened.

**Review by itykhan — APPROVED** [review]

**Review by sverb — APPROVED** [review]

**Pull request merged**